js/ui/tooltips/number-scrubber.js

```diff
@@ -79,6 +79,14 @@ TooltipEngine.classes.numberScrubber = TooltipBase.extend({
                     self.$el.addClass("dragging");
                     $(this).css("visibility", "hidden");
                     self.trigger("scrubbingStarted");
+                    // The text-to-be-tweaked needs to be the same length at the start and end
+                    // of the anti-undo changes.
+                    // I could probably just remember the length, but I like putting back the
+                    // original string. (It might even matter for i18n.)
+                    var Range = ace.require("ace/range").Range;
+                    var loc = self.aceLocation;
+                    var range = new Range(loc.row, loc.start, loc.row, loc.start + loc.length);
+                    self.originalString = parent.editor.getTextRange(range);
```

Comment:
Check that __bower_components/iframe-overlay/dist/iframe-overlay.js__ exists. My guess is that windows and/or git for windows is having troubles with the symlinks.  

You might try the first solution in http://stackoverflow.com/questions/5917249/git-symlinks-in-windows.  Alternately you could modify build-paths.json and replace __external/iframe-overlay/dist/iframe-overlay.js__ with __bower_components/iframe-overlay/dist/iframe-overlay.js__.  

I think the later is probably the better approach b/c it solves the problem for other windows users.